frontend/src/components/contexts/contexts.ts

```diff
@@ -9,4 +10,9 @@ interface ServicesInterfaces {
   authenticationHandlerService: AuthenticationHandlerService,
 }
 
-export const ServiceContext = React.createContext<undefined | ServicesInterfaces>(undefined);
+interface UserLoginStatusInterface {
+  loginStatusHandlerService: LoginStatusHandlerService,
+}
+
+export const ServiceContext = React.createContext<undefined|ServicesInterfaces>(undefined);
```

Comment:
What's the rationale for having two separate contexts, instead of just putting everything in ServiceContext? By its name it seems like LoginStatusHandlerService is indeed a service